From 290f404485e473ee67539ebbdd92691f20db3dc6 Mon Sep 17 00:00:00 2001
From: Kam Nasim <kam.nasim@windriver.com>
Date: Wed, 25 Oct 2017 11:32:54 -0400
Subject: [PATCH 06/14] Keystone Pike Rebase - WSGI

Reparent Keystone's launch script under etc/ to prevent autodoc tox
failures
---
 admin.py      | 22 ----------------------
 etc/admin.py  | 21 +++++++++++++++++++++
 etc/public.py | 21 +++++++++++++++++++++
 public.py     | 22 ----------------------
 4 files changed, 42 insertions(+), 44 deletions(-)
 delete mode 100644 admin.py
 create mode 100644 etc/admin.py
 create mode 100644 etc/public.py
 delete mode 100644 public.py

diff --git a/admin.py b/admin.py
deleted file mode 100644
index 91ec8e3..0000000
--- a/admin.py
+++ /dev/null
@@ -1,22 +0,0 @@
-# Copyright (c) 2013-2017 Wind River Systems, Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License"); you may
-# not use this file except in compliance with the License. You may obtain
-# a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-# License for the specific language governing permissions and limitations
-# under the License.
-
-
-from keystone.server import wsgi as wsgi_server
-
-import sys
-sys.argv = sys.argv[:1]
-
-if __name__ == "__main__":
-    application = wsgi_server.initialize_admin_application()
diff --git a/etc/admin.py b/etc/admin.py
new file mode 100644
index 0000000..548f1d2
--- /dev/null
+++ b/etc/admin.py
@@ -0,0 +1,21 @@
+# Copyright (c) 2013-2017 Wind River Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+from keystone.server import wsgi as wsgi_server
+
+import sys
+sys.argv = sys.argv[:1]
+
+application = wsgi_server.initialize_admin_application()
diff --git a/etc/public.py b/etc/public.py
new file mode 100644
index 0000000..d3a29f3
--- /dev/null
+++ b/etc/public.py
@@ -0,0 +1,21 @@
+# Copyright (c) 2013-2017 Wind River Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+from keystone.server import wsgi as wsgi_server
+
+import sys
+sys.argv = sys.argv[:1]
+
+application = wsgi_server.initialize_public_application()
diff --git a/public.py b/public.py
deleted file mode 100644
index ce39eec..0000000
--- a/public.py
+++ /dev/null
@@ -1,22 +0,0 @@
-# Copyright (c) 2013-2017 Wind River Systems, Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License"); you may
-# not use this file except in compliance with the License. You may obtain
-# a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-# License for the specific language governing permissions and limitations
-# under the License.
-
-
-from keystone.server import wsgi as wsgi_server
-
-import sys
-sys.argv = sys.argv[:1]
-
-if __name__ == "__main__":
-    application = wsgi_server.initialize_public_application()
-- 
2.7.4

